mapproxy/source/mapnik.py

```diff
@@ -118,6 +124,9 @@ def render_mapfile(self, mapfile, query):
     def _render_mapfile(self, mapfile, query):
         start_time = time.time()
 
+        if self.fonts:
+            mapnik.register_fonts(self.fonts)
```

Comment:
Can you move this to `__init__`? It should be enough to register each directory once.